app/support_packet.go

```diff
@@ -71,32 +71,67 @@ func (a *App) generateSupportPacketYaml() (*model.FileData, string) {
 	if err != nil {
 		return nil, errors.Wrap(err, "error while getting user count").Error()
 	}
 
+	analytics, err := a.GetAnalytics("standard", "")
+	if analytics == nil {
+		return nil, errors.Wrap(err, "error while getting analytics").Error()
+	}
+
+	elasticPostIndexing, _ := a.Srv().Store().Job().GetAllByTypePage("elasticsearch_post_indexing", 0, 2)
+	elasticPostAggregation, _ := a.Srv().Store().Job().GetAllByTypePage("elasticsearch_post_aggregation", 0, 2)
+	ldapSyncJobs, _ := a.Srv().Store().Job().GetAllByTypePage("ldap_sync", 0, 2)
+	messageExport, _ := a.Srv().Store().Job().GetAllByTypePage("message_export", 0, 2)
+	dataRetentionJobs, _ := a.Srv().Store().Job().GetAllByTypePage("data_retention", 0, 2)
+	complianceJobs, _ := a.Srv().Store().Job().GetAllByTypePage("compliance", 0, 2)
+	migrationJobs, _ := a.Srv().Store().Job().GetAllByTypePage("migrations", 0, 2)
+
+	licenseTo := ""
 	supportedUsers := 0
 	if license := a.Srv().License(); license != nil {
 		supportedUsers = *license.Features.Users
+		licenseTo = license.Customer.Company
 	}
 
 	// Creating the struct for support packet yaml file
 	supportPacket := model.SupportPacket{
-		ServerOS:              runtime.GOOS,
-		ServerArchitecture:    runtime.GOARCH,
-		ServerVersion:         model.CurrentVersion,
-		BuildHash:             model.BuildHash,
-		DatabaseType:          databaseType,
-		DatabaseVersion:       databaseVersion,
-		LdapVendorName:        vendorName,
-		LdapVendorVersion:     vendorVersion,
-		ElasticServerVersion:  elasticServerVersion,
-		ElasticServerPlugins:  elasticServerPlugins,
-		ActiveUsers:           int(uniqueUserCount),
-		LicenseSupportedUsers: supportedUsers,
+		LicenseTo:                  licenseTo,
+		ServerOS:                   runtime.GOOS,
+		ServerArchitecture:         runtime.GOARCH,
+		ServerVersion:              model.CurrentVersion,
+		BuildHash:                  model.BuildHash,
+		DatabaseType:               databaseType,
+		DatabaseVersion:            databaseVersion,
+		DatabaseSchemaVersion:      databaseSchemaVersion,
+		LdapVendorName:             vendorName,
+		LdapVendorVersion:          vendorVersion,
+		ElasticServerVersion:       elasticServerVersion,
+		ElasticServerPlugins:       elasticServerPlugins,
+		ActiveUsers:                int(uniqueUserCount),
+		LicenseSupportedUsers:      supportedUsers,
+		TotalChannels:              int(analytics[0].Value) + int(analytics[1].Value),
+		TotalPosts:                 int(analytics[2].Value),
+		TotalTeams:                 int(analytics[4].Value),
+		WebsocketConnections:       int(analytics[5].Value),
+		MasterDbConnections:        int(analytics[6].Value),
+		ReplicaDbConnections:       int(analytics[7].Value),
+		DailyActiveUsers:           int(analytics[8].Value),
+		MonthlyActiveUsers:         int(analytics[9].Value),
+		InactiveUserCount:          int(analytics[10].Value),
+		ElasticPostIndexingJobs:    elasticPostIndexing,
```

Comment:
I'd like these to be better formatted for YAML, it doesn't auto indent the arrays. If anyone has an idea on how to do that, I'm all ears!

Reply:
wondering if it would make sense to just drop these in as inline json

Reply:
I'd like to keep it still in the YAML format, so it can be parsed down the road easier. Changing up the format makes that hard.

Reply:
@coltoneshaw There is already an array field `ElasticServerPlugins` and I guess that is working fine. In any case, there is another way to format the array like this:
```go
B struct {
                RenamedC int   `yaml:"c"`
                D        []int `yaml:",flow"`
        }
```

You can refer the whole example in the doc: https://github.com/go-yaml/yaml/tree/v2

Reply:
Thanks! Will this change the way it's formatted in the YAML? I'll play with it